pkg/connections/ssh.go

```diff
@@ -26,26 +26,26 @@ type SSHConn struct {
 	client   *ssh.Client
 	reader   io.Reader
 	writer   io.WriteCloser
-	timeout  uint8
```

Comment:
`uint8`  is more suitable here, i used `int` in my example for the sake of the example. sorry for confusion.